src/tools/crawl.ts

```diff
@@ -1,20 +1,27 @@
+// Comment what this file does, and overall add more comment to every file and blocks of code.
+
 import * as fg from 'fast-glob';
 import * as fs from 'fs';
 import * as path from 'path';
 
 import { IGroupDesc } from '../framework/loader.js';
 
+// Could probably be a constant exposed by loader.js
 const specSuffix = '.spec.ts';
 
+// So this just returns the list of all nodes of the suite and their description but without the tree structure?
```

Comment:
Yes, and without the tests/cases either. For now.

Reply:
Ack